tsl/src/continuous_aggs/common.c

```diff
@@ -515,10 +515,12 @@ cagg_query_supported(const Query *query, StringInfo hint, StringInfo detail, con
 		return false;
 	}
 
-	if (query->hasWindowFuncs)
+	if (!ts_guc_enable_cagg_window_functions && query->hasWindowFuncs)
 	{
-		appendStringInfoString(detail,
-							   "Window functions are not supported by continuous aggregates.");
+		appendStringInfoString(detail, "Window function support not enabled.");
```

Comment:
Can we add details if window function support is enabled to mention that this is experimental which can produce unexpected results based on functions used?

Or something in the similar vein.

Reply:
+1